#! /bin/sh /usr/share/dpatch/dpatch-run
## 92_requantfactor.dpatch by FireFly
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixed requant factor (http://vdr-developer.org/mantisbt/view.php?id=95).

@DPATCH@
--- vdr-plugin-burn-0.1.0~pre21/jobs.c
+++ vdr-plugin-burn-0.1.0~pre21/jobs.c
@@ -304,10 +304,10 @@
 		size_pair::size_type diskFree =
 				size_pair::size_type( get_disk_size_mb() ) * MEGABYTE(1) - get_tracks_size( cut_, track_info::streamtype_audio );
 		size_pair::size_type videoSize =
-				size_pair::size_type( double( get_tracks_size( cut_, track_info::streamtype_video ) ) * 1.06 );
+				size_pair::size_type( double( get_tracks_size( cut_, track_info::streamtype_video ) ) * 1.04 );
 
 		return videoSize > diskFree
-				 ? double( get_tracks_size( cut_, track_info::streamtype_video ) ) / diskFree + .12
+				 ? double( videoSize ) / diskFree
 				 : 1;
 	}
 
